ProjectB/ProjectB/Services/IApiRequester.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Refit;
+using ProjectB.deserialize;
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using ProjectB.deserialize.HotelsFromJSON;
+using ProjectB.deserialize.HotelDetailsFromJSON;
+
+namespace ProjectB.Services
+{
+    public interface IApiRequester
```

Comment:
Suggest to use `IHotelsClient` name instead of `IApiRequester` and put this interface to `Clients` folder